src/js/components/Button/__tests__/Button-test.js

```diff
@@ -1,15 +1,31 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { cleanup, fireEvent, render } from '@testing-library/react';
+
 import 'jest-styled-components';
-import { Add, Next } from 'grommet-icons';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
 
+import { axe } from 'jest-axe';
+import { Add, Next } from 'grommet-icons';
+import { cleanup, fireEvent, render } from '@testing-library/react';
 import { findAllByType } from '../../../utils';
 import { Grommet, Button, Text } from '../..';
 
 describe('Button', () => {
   afterEach(cleanup);
 
+  test('should have no accessibility violations', async () => {
+    const { container, getByText } = render(
+      <Grommet>
+        <Button label="Test" onClick={() => {}} />
```

Comment:
would we be able to use `a11yTitle` here instead of label?

Reply:
I added the `a11yTitle` to the test but also left the `label="Test"` in there because they are doing two different things. The `a11yTitle` is for the aria-label and the `label="Test"` is for the label text that goes on the button